jobs/checkmarx-security-job-example.groovy

```diff
@@ -0,0 +1,14 @@
+import jenkins.automation.utils.CheckmarxSecurityJobBuilder
+
+def projectName ='foo'
+new CheckmarxSecurityJobBuilder(
+        name: "${projectName}checkmarx_security_job",
```

Comment:
let's use hyphens and not underscores, per our conventions for other jobs; let's also make sure there's a hyphen between projectname and the next stuff.

Also, "projectname-checkmarx-security-job" doesn't seem to match our conventions so far. Looking at our jenkins, it's simply "projectname-checkmarx"... any reason we wouldn't just stick with that?